conf/drupal/config/user.role.authenticated.yml

```diff
@@ -14,11 +14,11 @@ permissions:
   - 'access shortcuts'
   - 'access site-wide contact form'
   - 'access user profiles'
-  - 'create topic content'
   - 'edit own topic content'
   - 'post comments'
   - 'search content'
   - 'skip comment approval'
+  - 'use session_proposal transition create_new_draft'
```

Comment:
Should this user also have permissions for the `submitted_proposal`?  At the minute they can only create content in the "Draft" state.